Menu.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Saira:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/normalize.css" />
+    <link rel="stylesheet" href="./css/menubook.css" />
+    <link rel="stylesheet" href="./css/headerFooter.css" />
+
+    <title>Group Project Title</title>
+  </head>
+  <body>
+    <header class="header">
+      <div id="paw"></div>
+      <a href="#" id="aboutUs">About us</a>
+      <a href="#" id="menu">Menu</a>
+      <a href="#" id="careers">Careers</a>
+      <a href="#" id="buttonOrder">ORDER NOW</a>
+    </header>
+    <main>
+      <section class="menu-card">
+        <h1 class="menu-title">Our Menu</h1>
+        <section class="menu-card-selection">
+          <div class="nonalco-card">
+            <p class="selection-icon-title">Non-Alcohol</p>
+            <img src="./assets/img/nonAlcoholicBeveragesIntro.jpg" alt="" />
+          </div>
+          <div class="alcohol-card">
+            <p class="selection-icon-title">Alcohol</p>
+            <img src="./assets/img/alchohol.png" alt="" />
+          </div>
+          <div class="maindish-card">
+            <p class="selection-icon-title icon-title-maindish">Main</p>
+            <img src="./assets/img/main.jpg" alt="" />
+          </div>
+          <div class="desserts-card">
+            <p class="selection-icon-title icon-title-desserts">Desserts</p>
+            <img src="./assets/img/deserts.jpg" alt="" />
```

Comment:
Typo in desserts, rename jpg. Search engine checkers also check for spelling errors and will reduce a site's grade if it finds typos.